mentors/ruby/d.json

```diff
@@ -1,5 +1,4 @@
-[
-  {
+[{
```

Comment:
Try to avoid making changes that are outside of the scope of the PR.  The two unnecessary changes I have flagged are minor so I will go ahead and provide approval for this PR.

Reply:
Thanks. In the future, I will be aware of these.